app-config.yaml

```diff
@@ -57,6 +57,18 @@ proxy:
     changeOrigin: true
     secure: true
 
+  '/sonarqube':
+    target: https://sonarcloud.io/api
+
+    allowedMethods: ['GET']
+    # uncomment if using a local instnace of sonarqube
+    # auth: "${SONARQUBE_TOKEN}:"
+    # target: ${SONARQUBE_URL}/api
+
+  # uncomment if using a local instnace of sonarqube
+  # sonarQube:
+  #   baseUrl: ${SONARQUBE_URL}
```

Comment:
This can be removed as the backend is not installed

Reply:
it's needed based on the docs. https://roadie.io/backstage/plugins/sonarqube/#:~:text=sonarQube%3A%0A%20%20baseUrl%3A%20https%3A//your.sonarqube.instance.com

do you want me to remove it?